Problem1.py

```diff
@@ -0,0 +1,56 @@
+# Time Complexity :O(n^m) where n is number of coins and m is amount
+# Space Complexity :O(m)
+#  Did this code successfully run on Leetcode :yes
```

Comment:
This code did not run on leetcode for me. Please be very careful when you state if the code runs on leetcode. Please contact me, the cohort TA, or the professor if you need any help.